spec/serializers/category_list_serializer_extension.rb

```diff
@@ -3,7 +3,6 @@
 require "rails_helper"
```

Comment:
This file needs a `_spec.rb` ending, otherwise I don't think CI will pick it up when running specs.